src/screen/playing.rs

```diff
@@ -18,15 +18,12 @@ pub(super) fn plugin(app: &mut App) {
 
 fn enter_playing(mut commands: Commands) {
     commands.trigger(SpawnLevel);
+    commands.trigger(Soundtrack::Gameplay);
 }
 
-/// Reset the camera to the default position.
-/// Not needed in this case, as we don't actually move the camera, but useful
-/// for most games.
-fn exit_playing(mut cameras: Query<&mut Transform, With<Camera>>) {
-    for mut transform in &mut cameras {
-        *transform = default();
-    }
+fn exit_playing(mut commands: Commands) {
```

Comment:
I don't think camera transform reset should be removed. Regardless it shouldn't be removed as part of this PR.

Reply:
As mentioned in the comments, the camera is stationary in the example.
This is unnecessary

Reply:
If anything, I'd be more towards making a dedicated, permanent UI camera, and a gameplay camera that gets created at the beginning

Reply:
In the majority of games, the camera won't be stationary. And if it is, resetting its transform doesn't hurt. Multiple cameras seems unnecessary since the same camera can be used as the main game camera as well as the default UI camera.

Anyways this is all unrelated to audio.